Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -27,6 +27,17 @@ function Get-xSharePointAssemblyVersion() {
     return (Get-Command $PathToAssembly).FileVersionInfo.FileMajorPart
 }
 
+function Get-xSharePointServiceContext {
+    [CmdletBinding()]
+    param
+    (
+        [parameter(Mandatory = $true,Position=1)]
+        $proxyGroup
```

Comment:
Function parameter names should be Pascal case per [style guidance](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#parameter-names-should-use-pascalcase-)